nixos/doc/manual/administration/cleaning-store.xml

```diff
@@ -40,7 +40,7 @@ as old system configurations.  Thus they do not remove the ability to
 roll back to previous configurations.  The following command deletes
 old roots, removing the ability to roll back to them:
 <screen>
-$ nix-collect-garbage -d
+# nix-collect-garbage -d
```

Comment:
Hmm, well, with `-d` I'm not certain. You certainly need to own the profiles from which you want to remove roots, so you would most likely want to run it as root in this context (when removing OS generations).